index.html

```diff
@@ -234,10 +235,10 @@
       <li>Working with vectors and data frames</li>
       <li>Reading and plotting data</li>
       <li>Creating and using functions</li>
-      <li>Loops and conditionals</li>
-      <li>Using R from the command line</li>
```

Comment:
We have to close the pull request because I need to delete this entire section - it's apparently a leftover from the Software Carpentry template. I have not seen any Data Carpentry workshop pages use this syllabus section. The main reason DC pages do not use this section is because it does not describe the Data Carpentry material. These topics are the Software Carpentry workshop topics.  

> On Apr 3, 2016, at 21:52, Dirk Eddelbuettel <notifications@github.com> wrote:
> 
> In index.html:
> 
> > @@ -234,10 +235,10 @@
> >        <li>Working with vectors and data frames</li>
> >        <li>Reading and plotting data</li>
> >        <li>Creating and using functions</li>
> > -      <li>Loops and conditionals</li>
> > -      <li>Using R from the command line</li>
> I still don't see the point of removing this. These are valid R topics. Whether we cover them or not.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>